schema_based_on_diagram.sql

```diff
@@ -0,0 +1,53 @@
+CREATE TABLE medical_histories (
+  id SERIAL PRIMARY KEY,
+  admitted_at TIMESTAMP NOT NULL,
+  patient_id INTEGER NOT NULL,
+  status VARCHAR(255) NOT NULL,
+  FOREIGN KEY (patient_id) REFERENCES patients(id)
+);
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY, 
+  name VARCHAR(250) NOT NULL, 
+  date_of_birth DATE) NOT NULL;
+
+CREATE TABLE invoices(
+  id SERIAL PRIMARY KEY, 
+  total_amount DECIMAL(5, 2) NOT NULL, 
+  generated_at TIMESTAMP NOT NULL, 
+  payed_at TIMESTAMP NOT NULL, 
+  medical_history_id INT NOT NULL
+  )
+
+create table treatments (
+  id SERIAL PRIMARY KEY,
+  type VARCHAR(250) NOT NULL,
+  name VARCHAR(250) NOT NULL,
+)
+
+create table invoice_items (
+  id primary key,
+  unit_price DECIMAL(5, 2) NOT NULL,
+  quantity INT NOT NULL,
+  total_price DECIMAL(5, 2) NOT NULL,
+  invoice_id INT NOT NULL,
+  treatment_id INT NOT NULL
+)
+
+
```

Comment:
As it´s not shown on the diagram, there´s one missing table that is created from the relationship between treatments and medical_histories tables.

- Kindly create this table, maybe you can name it treatments_histories 💯